recipe/meta.yaml

```diff
@@ -12,31 +12,24 @@ source:
   url:
     - https://cran.r-project.org/src/contrib/Rserve_{{ version }}.tar.gz
     - https://cran.r-project.org/src/contrib/Archive/Rserve/Rserve_{{ version }}.tar.gz
-
-
   sha256: f6d636c736c3f16487d9987e54554fe0c55257b9bc0671b43e536d832e513027
 
 build:
   merge_build_host: True  # [win]
-  number: 1
-
+  number: 1001
   rpaths:
     - lib/R/lib/
     - lib/
-  skip: True  # [win32]
 
 requirements:
   build:
     - posix                # [win]
     - make
-    - {{ compiler('c') }}        # [unix]
-    - {{ compiler('m2w64_c') }}  # [win]
-    - {{ compiler('cxx') }}        # [unix]
-    - {{ compiler('m2w64_cxx') }}  # [win]
```

Comment:
Why was this changed?

Reply:
We still need the m2 compilers for windows